src/equations/numerical_fluxes.jl

```diff
@@ -221,6 +221,46 @@ See [`FluxLaxFriedrichs`](@ref).
 """
 const flux_lax_friedrichs = FluxLaxFriedrichs()
 
+"""
+    DissipationEntropyStable(max_abs_speed=max_abs_speed_naive)
+
+Create a local Lax-Friedrichs-type dissipation operator that is provably entropy stable. This operator
+must be used together with an entropy-conservative two-point flux function (e.g., `flux_ec`) to yield 
+an entropy-stable surface flux. The surface flux function can be initialized as:
+```
+flux_es = FluxPlusDissipation(flux_ec, DissipationEntropyStable())
+```
+
+In particular, the numerical flux has the form
+```math
+f^{ES} = f^{EC} + \frac{1}{2} λ_{max} H (w_r - w_l),
```

Comment:
```suggestion
f^{\mathrm{ES}} = f^{\mathrm{EC}} + \frac{1}{2} λ_{max} H (w_r - w_l),
```

Reply:
Does this render correctly? Previously, I used `\lambda` instead of λ, but the documenter failed to render it. To avoid issues, I rewrote the expression in a simpler form...

Reply:
As expected, the documenter fails...
<img width="1203" alt="image" src="https://github.com/user-attachments/assets/530e4fb2-5f2c-47c2-a6ac-a1e4291e40f8" />

Reply:
I reverted the commit.

Reply:
Hm, strange - Mathjax renders this correctly:

```math
f^{\mathrm{EC}}
```

Reply:
Ah I remember: I think you need double backslash if you are not in a @doc raw environment:

https://github.com/trixi-framework/Trixi.jl/blob/d01e4a7cf1c6d5424e27056511ce4c6ac592119d/src/equations/laplace_diffusion_3d.jl#L4-L5

Reply:
This should use `- \frac{1}{2}` in the dissipation term whatever is decided with the other formatting.